panel/config.py

```diff
@@ -115,6 +116,9 @@ class _config(_base_config):
     autoreload = param.Boolean(default=False, doc="""
         Whether to autoreload server when script changes.""")
 
+    browser_info = param.Boolean(default=True, doc="""
```

Comment:
I think it would be better to be off by default.

Reply:
Can see that argument but can you explain your reasoning? One of the main reasons we want to use it is for things like the device pixel ratio to determine the resolution of datashader plots and hardly anyone will know to enable this.

Reply:
The reason for the suggestion was that there was no reason in this PR why it should be on by default. 

You gave me two reasons.